.changeset/brave-lights-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": minor
+---
+
+Add template substitution
```

Comment:
Can you provide a bit more information here. Also `template` is a somewhat overloaded concept in C3. For example this could be read as substituting whole templates.

Reply:
Also can you comment that from now any template that has `__WORKER_NAME__` strings in wrangler.jsonc files will be converted to the name of the project.

Should we also be updating the name in the packge.json too?

Reply:
Added more info